neat.css

```diff
@@ -1,3 +1,7 @@
+:root {
+    color-scheme: dark light;
```

Comment:
This is one of those "opinionated" things, and it's a bit silly, but I wonder if you would swap this to `light dark`? Personally, I prefer `light`. *Looks around the room for shameful glances.*